src/containers/Login.tsx

```diff
@@ -4,6 +4,7 @@ import { TenantSelector } from '@cognite/gearbox';
 import styled from 'styled-components';
 import { bindActionCreators, Dispatch } from 'redux';
 import { setTenant } from '../modules/app';
+import { sdk } from '..';
```

Comment:
Hmm?

Reply:
hehe ill add `../index` to make it easier to read